snippets/service-account-auto-secret-removed.adoc

```diff
@@ -0,0 +1,17 @@
+// When including this file, ensure that {FeatureName} is set immediately before
+// the include. Otherwise it will result in an incorrect replacement.
+
+[id="auto-generated-sa-token-secrets_{context}"]
+== About automatically-generated service account token secrets
+
+In {product-version}, {product-title} is adopting an link:https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.24.md#urgent-upgrade-notes-1[enhancement from upstream Kubernetes] where the `LegacyServiceAccountTokenNoAutoGeneration` feature is enabled by default. As a result, when creating new serivce accounts (SA), a service account token secret is no longer automatically genereated. Previously, {product-title} automatically added a service account token to a secret for each new SA. 
+
+However, some features and workloads need service account token secrets to communicate with the Kubernetes API server, for example, the OpenShift Controller Manager. While this requirement will be changed in a future release, it remains in {product-title} {product-version}. As a result, if you need a service account token secret, you must manually use the TokenRequest API to request bound service account tokens or create a service account token secret. 
+
+After upgrading to {product-version}, existing service account token secrets are not deleted and continue to function as expected.
+
+[NOTE]
+====
+In {product-version}, service account token secrets still appear to have been automatically generated. Although, instead of the current creation two secrets per service account, there is now only one. That token does not work. In a future release, the number will be further reduced to zero. Note that `dockercfg` secrets are still generated and no secrets are deleted during upgrades.
```

Comment:
"instead of the current creation two secrets per service account" sounds weird. Should it say "Instead **of** the current creation of two secrets per account?"

"That token does not work" which token is this referring to?